include/zephyr/drivers/lora.h

```diff
@@ -66,6 +66,21 @@ struct lora_modem_config {
 
 	/* Set to true for transmission, false for receiving */
 	bool tx;
+
+	/* Invert the IQ. Normally this should be set to false. In advanced
+	 * use-cases where a differentation is needed between "uplink" and
+	 * "downlink" traffic, the IQ can be inverted to create two different
+	 * channels on the same frequency */
+	bool iq_inverted;
+
+	/* Sets the sync-byte to use:
+	 * false for using the private network sync byte
+	 * true for using the public network sync byte
+	 * true is only intended for advanced usage. Normally false should be used
```

Comment:
I would document this in terms of public/private not true/false. For example:
`The public network sync-byte is only intended for advanced usage.`